### DISCUSSION TITLE
Random Wait Function

### DISCUSSION THREAD

**Discussion by didiersm** [pybricks/support#1306] [General]
HI
Can we use a random seconds number with Pybricks
Many Thks

**Comment by laurensvalk** [comment]
```python

from urandom import randint
from pybricks.tools import wait

# Wait between 500 and 2000 milliseconds
wait(randint(500, 2000))

# Or if you need much longer, for example 3 -- 15 seconds
wait(randint(3, 15) * 1000)

```

**Reply by didiersm** [reply]
Thanks it work perfectly.
It missed me the randint import

**Reply by EmperorKabir** [reply]
Is this available in the block editor @laurensvalk ?

**Reply by laurensvalk** [reply]
Yes! Here is an example of blinking the light on and off for random durations, so it looks a bit like the "busy writing" indicator on old hard disks :computer: 

![image](https://github.com/pybricks/support/assets/12326241/2df26159-c911-4566-9231-6f2091843482)

**Reply by EmperorKabir** [reply]
A helpful answer with bonus nostalgia! Thank you!

**Reply by laurensvalk** [reply]
I may have gotten the nostalgia part wrong though. Maybe it meant busy reading? I don't remember :smile:
